client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -89,9 +89,24 @@ export const OutboundLineEdit = ({
 
   const handleSave = async (onSaved: () => boolean | void) => {
     const confirmZeroQuantityMessage = t('messages.confirm-zero-quantity');
+    const unsavedVvmStatusChange = t('messages.unsaved-outbound-vvm-status');
+    const vvmStatusChanged = draftLines.some(line => {
+      const originalId = line.vvmStatusId ?? null;
+      const currentId = line.vvmStatus?.id ?? null;
+      return currentId !== originalId;
+    });
+
+    if (
+      vvmStatusChanged &&
+      !alerts.some(alert => alert.message === unsavedVvmStatusChange)
```

Comment:
should only show this alert if there line has no allocated quantity for that line right?

https://github.com/user-attachments/assets/7fb3710e-030f-4ab9-a125-8ad3509ee7cb